### PR TITLE
Update buy-bitclout-usd.component.html

### PR DESCRIPTION
**Pull request by ShhubhamA**
2nd PR  minor html change of bitclout to $clout inspired by the one who submitted last PR  :)😛😛

### CODE REVIEW

**Comment by CoderDrShen** [comment]
I am working on Turkish Node. That's why I wanted to fix minor mistakes.

File: src/app/buy-bitclout-page/buy-bitclout-usd/buy-bitclout-usd.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="w-100">
   <div class="px-15px">
-    <div class="fs-18px font-weight-bold mb-10px">Enter an amount in USD to convert to BitClout</div>
-    <p class="mb-5px">You can purchase up to ${{ maxUsdAmount }} of BitClout per week using USD.</p>
+    <div class="fs-18px font-weight-bold mb-10px">Enter an amount in USD to convert to $Clout</div>
```
Comment (lazynina) [inline src/app/buy-bitclout-page/buy-bitclout-usd/buy-bitclout-usd.component.html:3]:
```suggestion
    <div class="fs-18px font-weight-bold mb-10px">Enter an amount in USD to convert to $CLOUT</div>
```

File: src/app/buy-bitclout-page/buy-bitclout-usd/buy-bitclout-usd.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+    <p class="mb-5px">You can purchase up to ${{ maxUsdAmount }} of $Clout per week using USD.</p>
```
Comment (lazynina) [inline src/app/buy-bitclout-page/buy-bitclout-usd/buy-bitclout-usd.component.html:4]:
```suggestion
    <p class="mb-5px">You can purchase up to ${{ maxUsdAmount }} of $CLOUT per week using USD.</p>
```

File: src/app/buy-bitclout-page/buy-bitclout-usd/buy-bitclout-usd.component.html [hunk 3]
```diff
@@ -72,7 +72,7 @@
       (click)="onBuyClicked()"
       [ngClass]="{ disabled: quotationError }"
     >
-      Buy BitClout
+      Buy $Clout
```
Comment (lazynina) [inline src/app/buy-bitclout-page/buy-bitclout-usd/buy-bitclout-usd.component.html:75]:
```suggestion
      Buy $CLOUT
```

**Review by lazynina — CHANGES_REQUESTED** [review]

**Review by lazynina — APPROVED** [review]

**Pull request merged**